test/AngularSpec.js

```diff
@@ -1861,6 +1861,44 @@ describe('angular', function() {
     });
   });
 
+  describe('isError', function() {
+    it('should not assume objects are errors', function() {
+      expect(isError({ message: 'A fake error', stack: 'no stack here'}))
+        .toBe(false);
+    });
+
+    it('should detect simple error instances', function() {
+      expect(isError(new Error())).toBe(true);
+    });
+
+    function testError(createError) {
```

Comment:
Move this either at the top or at the bottom of the `describe` block (or even create a nested `describe` for "other-context" tests) and give it a more descriptive name (e.g. `testErrorFromDifferent context`.

Maybe having a separate `describe` block with `beforeAll`/`afterAll` blocks for creating/destorying the iframe is a good idea.